### PR TITLE
[APB-4193][PT]: Update IV Pages

### PR DESCRIPTION
**Pull request by JTY009**

### CODE REVIEW

File: app/uk/gov/hmrc/agentsexternalstubsfrontend/connectors/AgentsExternalStubsConnector.scala [hunk 1]
```diff
@@ -198,4 +199,20 @@ class AgentsExternalStubsConnector @Inject()(
       .recover {
         case Upstream4xxException(e) => throw e
       }
+
+  def createJourneyId(serviceContract: ServiceContract, journeyType: JourneyType)(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext): Future[String] = {
+    val json = Json.toJson(serviceContract).as[JsObject] ++ Json.obj("journeyType" -> journeyType)
+    println(s"So $json")
```
Comment (mariannepearson0) [inline app/uk/gov/hmrc/agentsexternalstubsfrontend/connectors/AgentsExternalStubsConnector.scala]: print line

Comment (mariannepearson0) [reply app/uk/gov/hmrc/agentsexternalstubsfrontend/connectors/AgentsExternalStubsConnector.scala]: is having a database in the backend for this potentially a little bit of overkill? If the journey ID is only used to get the outcome reason of a particular journey then we could just have a Map of reason -> code like Map("LockedOut" -> "lockedout123", ...). It's only a stub, what do you think?

File: app/uk/gov/hmrc/agentsexternalstubsfrontend/controllers/IdentityVerificationController.scala [hunk 2]
```diff
@@ -28,60 +32,124 @@ class IdentityVerificationController @Inject()(
...
     }
 
+  private def redirectWithJourneyId(targetUrl: String, journeyId: String, option: String): Result = {
+    val cleanUrl = URLDecoder.decode(targetUrl, "UTF-8")
+    val tokenAppendChar = if (cleanUrl.contains("?")) "&" else "?"
+    Redirect(s"$cleanUrl${tokenAppendChar}journeyId=$journeyId")
+      .withHeaders("X-Result-Location" -> s"/mdtp/journey/journeyId/$option")
+  }
+
+  def getIvResult(journeyIdAndReason: String): Action[AnyContent] = Action.async { implicit request =>
```
Comment (mariannepearson0) [inline app/uk/gov/hmrc/agentsexternalstubsfrontend/controllers/IdentityVerificationController.scala]: do we need to have a getIveResultInternal for staging?

Comment (JTY009) [reply app/uk/gov/hmrc/agentsexternalstubsfrontend/controllers/IdentityVerificationController.scala]: Ah fair point

**Pull request merged**